org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4graph/Graph.java

```diff
@@ -37,6 +37,8 @@
  * Mutable; represents a graph.
  * <p>
  * <b>Thread Safety:</b> Can be called only by the AWT event thread.
+ *
+ * @modified Nuno Macedo // [HASLab] electrum-base
```

Comment:
this change is not really Electrum related, but rather a need we had to have atoms of the same type ordered by identifier within the same layer of the graph

Reply:
not on my side. I started this practice mainly to keep track of changes by some students, and then kept it because it could be useful during this PR, but I'm ok with removing them